### ISSUE TITLE
Any plans to build the app for MacOS and iOS, perhaps?

### ISSUE THREAD

**Issue by GregTonoski** [moparisthebest/seedxor#1]

**Comment by moparisthebest** [comment]
It should build just fine for MacOS, I just don't have one to build on.

It's a command-line application though, can iOS run those?

**Comment by GregTonoski** [comment]
> It's a command-line application though, can iOS run those?

No, iOS doesn't run command-line applications. I think about .ipa that could be distributed via TestFlight (and AppStore ultimately).
